PIL/Image.py

```diff
@@ -1129,7 +1129,8 @@ def _crop(self, im, box):
 
     def draft(self, mode, size):
         # type: (Mode, Size) -> Any
-        # TODO: return type clarification
+        # FIXME TYPING: return type clarification
+        # FIXME TYPING: thumbnail calls as (None, size), so is it Optional[Mode]?
```

Comment:
Have made the Optional change, but have left a comment for the Any/return-type based on your response, at least for now.